src/main/java/seedu/address/model/util/UserTypeUtil.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.model.util;
+
+import seedu.address.model.person.UserType;
+
+/**
+ * Utility methods related to Collections
+ */
+
+public class UserTypeUtil {
+
+    /** Returns a "buyer" String */
+    private static String isBuyer() {
+        return "buyer";
+    }
+
+    /** Returns a "seller" String */
+    private static String isSeller() {
+        return "seller";
+    }
```

Comment:
Maybe use constant Strings instead since the method name sounds like it should return a boolean